data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -782,7 +836,11 @@ Please report the bug."
             name=check_income
         [/fire_event]
 
-        {TALK_ABOUT_LOC 11,14 ( _ "Remember to retreat your wounded units to villages. Healers can only heal 4 hitpoints at a time, while villages can heal 8 (the maximum healing for any unit).")}
+        {TALK_ABOUT_LOC 16,9 ( _ "Remember to retreat your wounded units to villages. Villages heal units faster than Elvish Shamans do: a unit in a village will be healed 8 hitpoints per turn, whereas a unit next to a Shaman will only be healed 4 points per turn.
+
+Different sources of healing are generally not combined. If a unit starts its turn next to multiple healers, or on a village and next to a healer, it will still be healed only once.
+
+Rest-healing is an exception to the rule — if a unit doesn’t do anything for a turn, it will heal 2 hitpoints in addition to any healing it may receive from villages or healers, and this does combine for a total of up to 10 hitpoints per turn.")}
```

Comment:
```suggestion
Rest-healing is an exception to the rule — if a unit doesn’t do anything for a turn, it will heal 2 hitpoints in addition to any healing it may receive from villages or healers, combining for a total of up to 10 hitpoints per turn.")}
```